### DISCUSSION TITLE
rule/texte/BR-BT-00137-214

### DISCUSSION THREAD

**Discussion by EMDUPONT** [OP-TED/eForms-SDK#787] [Q&A] [answered]
Hello,

We have some issues to published a notice of modification : when we want to publish, the plateform asks to convert the notice and this error message appears: rule/texte/BR-BT-00137-214

What do we need to change in order to publish?

Thank you

**Comment by YvesJo** [comment, marked as answer]
Hi,
I've converted the "issue" to a discussion as it is not a SDK issue.
Also for concerns with the XML notices, it is recommended to get in touch with the helpdesk directly ted@publications.europa.eu , providing at the same time further details like the SDK version.
Regarding the rule that led to an error: "For any Notice of Subtype 7 to 11, 15 to 19 or 22, any lot must have one of the followings: Duration Period (BT-36-Lot), Duration Start & End Dates (BT-536-Lot, BT-537-Lot), or Duration Other (BT-538-Lot)." there is at least one lot in the XML that is missing duration information.

**Reply by MathildeDI** [reply]
Hello @EMDUPONT
I have the same error and can't find the meaning. Did you get an answer from the helpdesk ?
Regards,
Mathilde

**Reply by YvesJo** [reply]
Hi Mathilde,
The same error is generally triggered by the same reason, there is most likely a Lot in your notice for which the Duration information has not been specified. You may also get in touch with the helpdesk for your specific situation, providing them all relevant information.
KR

**Reply by MathildeDI** [reply]
Hello @YvesJo in the meantime I was looking at your previous answer and found indeed that duration was missing in my notice. Thanks a lot for your help again :-)

**Comment by JanEmde** [comment]
@YvesJo Hi, I found this old thread. Wouldn't it make much more sense to attach the rule to the GR-Lot-PlannedDuration group? You would then see directly where the problem lies.
